src/dragon.rs

```diff
@@ -0,0 +1,37 @@
+/// Original dragon image modified from:
+/// https://www.rawpixel.com/image/581164/hand-drawn-dragon
+/// using the following webiste:
+/// https://www.text-image.com/convert/ascii.html
+pub fn loading(){
+print!("                                  `````           
+                                  `-/yyy:`        
+       `.:/:`                      /yddddy-       
+       ``.sss:`  ` ``            `/dmdddddh.      
+          +yys+++o:s/-::.`      :odmdhhyhhhy.     
+          `/osyyddddhhys:/--`  -yhdhhyyyyyyho`    
+          .o+yyhhdddhhyyhyso/./hhhsysosssosyy+    
+        `-ydddmddhhhyyyyshddhhmddh++oo+o+oooyy:   
+      `+yddhdhddsohhhhdyyhddmmddhso:/o//+/++oys`  
+    `-sddhhhhyyy+/o+yhyddhdddmddho+o::/:///+/+s.  
+   .hhyhyhdhys++:/ysysoyhhddhyhmho////+/:+++/++`  
+    -:-.+s+//.    ``- `hyyhhyshmd-  . :/:+o+/+/   
+      -ss:`           `:hy+oyhdho:     ` -/++++   
+      `..              `ydsoydmd+`        `.s++   
+                       `-.`./sdd:`          o:/   
+                            `/odo.          o..   
+                             :+do.         .--    
+                             -od:          ` `    
+                            `/ho`                 
+               `:-..```   `-oy+`                  
+             `/syhyyso++/+so/``                   
+            .syyys---::/--`                       
+           `/-.                                                                     
+           _         _           _ 
+  ___ __ _| | ___ __| |_ __   __| |
+ / __/ _` | |/ __/ _` | '_ \\ / _` |
+| (_| (_| | | (_| (_| | | | | (_| |
+ \\___\\__,_|_|\\___\\__,_|_| |_|\\__,_|
```

Comment:
maybe one thats a bit more readable?

Reply:
escape '\'  are what's causing this to look odd, without them it won't compile. It prints out as below. 
![image](https://user-images.githubusercontent.com/35909846/147512911-4133d27e-3ccd-4864-a55c-0b43b01c8704.png)

Reply:
Ahhhhhh!